crates/turbopack-node/src/lib.rs

```diff
@@ -249,10 +250,25 @@ enum EvalJavaScriptOutgoingMessage<'a> {
 #[derive(Deserialize)]
 #[serde(tag = "type", rename_all = "camelCase")]
 enum EvalJavaScriptIncomingMessage {
-    FileDependency { path: String },
-    BuildDependency { path: String },
-    DirDependency { path: String, glob: String },
-    JsonValue { data: String },
+    FileDependency {
+        path: String,
+    },
+    BuildDependency {
+        path: String,
+    },
+    DirDependency {
+        path: String,
+        glob: String,
+    },
+    JsonValue {
+        data: String,
+    },
+    EmittedError {
+        severity: IssueSeverity,
+        name: String,
+        message: String,
+        stack: Vec<StackFrame>,
```

Comment:
Nit: these 3 fields should just use the `StructuredError` struct so we can share deserialize code.